### PR TITLE
doc/authentication: clean up PKI instructions

### PR DESCRIPTION
**Pull request by ru-fu**
Move the information from the note about using PKI with ACME to the main instructions, and update the docs to cover a cluster setup as well.

### CODE REVIEW

File: doc/authentication.md [hunk 1]
```diff
@@ -108,20 +108,39 @@ Alternatively, the clients can provide the token directly when adding the remote
...
+The steps for enabling PKI mode differ slightly depending on whether you use an ACME provider in addition (see {ref}`authentication-server-certificate`).
 
-1. Add the {abbr}`CA (Certificate authority)` certificate to all machines:
+`````{tabs}
+````{group-tab} Only PKI
+If you use a PKI system, both the server certificates and the client certificates are issued by a secondary CA.
+
+1. Add the CA certificate to all machines:
    - Place the `client.ca` file in the clients' configuration directories (`~/.config/lxc` or `~/snap/lxd/common/config` for snap users).
    - Place the `server.ca` file in the server's configuration directory (`/var/lib/lxd` or `/var/snap/lxd/common/lxd` for snap users).
```
Comment (ru-fu) [inline doc/authentication.md:122]: The (now removed) note mentioned `server.ca` **and** `ca.crl`. Are we missing information about `ca.crl` here?

Comment (ru-fu) [inline doc/authentication.md:122]: In a cluster setup, would this file be called `cluster.ca`? Or is the current description valid for both single server and cluster setup?

Comment (markylaing) [reply doc/authentication.md:122]: This description is only valid for standalone LXD. For clustered LXD, it should be `cluster.ca` (plus `cluster.crt` and `cluster.key` in the lines below) and all cluster members must have the same certificates.

Also, while the `{server,cluster}.ca` must be present before LXD starts, the `{server,cluster}.{crt,key}` files can be updated at runtime using `lxc cluster update-certificate`.

Comment (markylaing) [reply doc/authentication.md:122]: The CRL is mentioned in the last paragraph of the section. Unlike the distinction between `server.ca` and `cluster.ca`, the `ca.crl` is the same whether in standalone or clustered modes.

**Comment by ru-fu** [comment]
Thanks @markylaing ! Updated to reflect that information.

**Review by markylaing — APPROVED** [review]
LGTM thanks :)

**Pull request merged**